README.md

```diff
@@ -1,8 +1,8 @@
-# Falcosidekick
+# Kubearmor-Falcosidekick
```

Comment:
Lets remove Falco from the name and add a credit line below.

Credits: This work is primarily a derivative work of falcosidekick.